### PR TITLE
Add Page Loader fixes #92

### PR DESCRIPTION
**Pull request by omar-sherif9992**

### CODE REVIEW

**Comment by omar-sherif9992** [comment]
I am done I'm waiting for your approval
I am Omar Sherif | Contributor from GSSOC22

**Review by BRAVO68WEB — APPROVED** [review]
OK but could  have added a custom loader

**Comment by BRAVO68WEB** [comment]
@allcontributors please add @omar-sherif9992 for code

**Pull request merged**
